daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/PackedBinaryTraits.scala

```diff
@@ -26,6 +26,8 @@ import org.apache.daffodil.lib.equality.TypeEqual
 import org.apache.daffodil.lib.exceptions.Assert
```

Comment:
We should probably hold off until https://github.com/apache/daffodil/pull/1337 is merged, since that touches some of this code and is the bigger commit, easier to deal with conflicts with the smaller commit.